packages/inter-protocol/test/smartWallet/oracle-integration.test.js

```diff
@@ -1,8 +1,7 @@
 import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
 
-import { NonNullish } from '@agoric/assert';
+import { NonNullish, zip } from '@agoric/internal';
```

Comment:
Aside, to prevent bundle bloat, consider refactoring `@agoric/internal` to expose `@agoric/internal/zip.js` and `@agoric/internal/non-nullish.js` so this code entrains only the parts it needs.

Reply:
agoric/internal allows deep imports, since it has no public API.

so this could be `import { zip } from '@agoric/internal/src/utils.js'

Reply:
If we're going to do this at all, I like the way we did it in @endo/common, where each was its own explicit top level export. I do think that it would be good to eventually do that here. But right now @agoric/internal's package.json doesn't even have an `"exports":` clause, in which case something like the above would be an unstructured deep import. 

So I'm resolving this in its current state.